.github/workflows/release.yml

```diff
@@ -0,0 +1,27 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+'
+
+jobs:
+  release:
+    name: Create Release
+    if: github.event.base_ref == 'refs/heads/master'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get Version
+        id: get_version
+        run: echo ::set-output name=version::${GITHUB_REF/refs\/tags\//}
+
+      - name: Create & Publish Release
+        uses: release-drafter/release-drafter@v5.12.1
+        with:
+          config-name: release-drafter-config.yml
+          name: ${{ steps.get_version.outputs.version }}
+          tag: ${{ steps.get_version.outputs.version }}
```

Comment:
Should this `tag` line differ from the previous `name` line and look like `${{ steps.get_tag.outputs.tag }}` instead?
The following [example](https://github.com/marketplace/actions/get-latest-tag) fetches the latest tag using: `${{ steps.previoustag.outputs.tag }}`, but I'm still new to this tool though.

Reply:
I'm not using the 'Get Latest Tag' action. That operates on the git repository. I simplified this task to just operate on GitHub event metadata. My custom `get_version` task is correct for the name, tag, and version. In our case, we're using the same value for all 3. I'm passing them explicitly as parameters to avoid using the Release Drafter's auto version increement.